src/render_tree.rs

```diff
@@ -2,8 +2,9 @@ use std::borrow::BorrowMut;
 use std::{cell::RefCell, rc::Rc};
 
 use crate::html5::node::NodeData;
-use crate::html5::parser::document;
+use crate::html5::parser::document::{self};
```

Comment:
```suggestion
use crate::html5::parser::document;
```

Reply:
This was a weird auto import from Zed, guess I didn't catch it